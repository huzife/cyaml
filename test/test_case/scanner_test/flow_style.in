---
flow_map: {
    key1: value1,
    ,
    key2: value2
}
flow_seq: [
    "1",
    2,
    ,
    3
]
...
