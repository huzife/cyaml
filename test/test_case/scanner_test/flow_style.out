(DOC_START)#
(BLOCK_MAP_START)#
(KEY)#
(SCALAR, flow_map)#
(VALUE)#
(FLOW_MAP_START)#
(KEY)#
(SCALAR, key1)#
(VALUE)#
(SCALAR, value1)#
(FLOW_ENTRY)#
(FLOW_ENTRY)#
(KEY)#
(SCALAR, key2)#
(VALUE)#
(SCALAR, value2)#
(FLOW_MAP_END)#
(KEY)#
(SCALAR, flow_seq)#
(VALUE)#
(FLOW_SEQ_START)#
(SCALAR, 1)#
(FLOW_ENTRY)#
(SCALAR, 2)#
(FLOW_ENTRY)#
(FLOW_ENTRY)#
(SCALAR, 3)#
(FLOW_SEQ_END)#
(BLOCK_MAP_END)#
(DOC_END)#
